bindings/generator/generate.py

```diff
@@ -580,15 +582,17 @@ def generate(what: str, api_specs: ApiSpecs) -> list[str]:
         api_module = import_module("api")
     api_specs = generate_api_specs(api_module)
 
-    rust_modified_packages = []
+    rust_modified_crates = []
 
     if "all" in args.what:
         args.what = TEMPLATE_CHOICES
 
     for what in args.what:
-        rust_modified_packages.extend(generate(what, api_specs))
+        possible_modified_crate = generate(what, api_specs)
+        if possible_modified_crate:
+            rust_modified_crates.append(possible_modified_crate)
```

Comment:
Why the change here? the `extend` method already manages empty lists

Reply:
Yes but `generate` return an `str | None` not a `list[str]`